145_Binary_Tree_Postorder_Traversal.py

```diff
@@ -0,0 +1,14 @@
+# O(n)
+class Solution:
+    def postorderTraversal(self, root: Optional[TreeNode]) -> List[int]:
+        def loop(root):
+            if root is None:
+                return 
```

Comment:
Insert a blank line after the return statement

Reply:
- I insert a blank line after the return statement